_format/1.1/index.md

```diff
@@ -190,8 +200,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### <a href="#document-resource-object-identification" id="document-resource-object-identification" class="headerlink"></a> Identification
 
-Every [resource object][resource objects] **MUST** contain an `id` member and a `type` member.
-The values of the `id` and `type` members **MUST** be strings.
+Every [resource object][resource objects] **MUST** contain an `id` member and a
+`type` member. As noted above, the only exception is that the `id` member is not
```

Comment:
I think this is confusing phrasing. It means we've got a sentence that's false on its own, and needs to be taken together with another sentence. Can we find something that doesn't have to contradict itself later? Maybe something like:
> Every resource object **MUST** contain a type member. Every resource object **MUST** contain an id member, except when the resource object originates at the client and represents a new resource.

Reply:
> It means we've got a sentence that's false on its own, and needs to be taken together with another sentence. Can we find something that doesn't have to contradict itself later? 

I agree this is a best practice and will reword. Thanks for the suggestion!